zookeeper-server/src/test/java/org/apache/zookeeper/test/AtomicFileOutputStreamTest.java

```diff
@@ -64,7 +66,7 @@ public void testWriteNewFile() throws IOException {
         fos.close();
         assertTrue(dstFile.exists());
 
-        String readBackData = ClientBase.readFile(dstFile);
+        String readBackData = FileUtils.readFileToString(dstFile, StandardCharsets.UTF_8);
```

Comment:
Also, there are many new ways in Java 7/8 to read from a file. I'm not sure it is necessary to rely on Commons for this. Keeping the number of uses of Commons down means that it may be easier to drop as a dependency in future, as functions are provided directly by Java itself. Fewer dependencies means greater stability in the long run, so if ZK doesn't need to use a dependency, it shouldn't.

Here are some ways to read from a File without commons, found by using Google: https://howtodoinjava.com/java/io/java-read-file-to-string-examples/

We probably want something like:

```java
  String readBackData = new String(Files.readAllBytes(dst.toPath()), UTF_8);
```

Reply:
Good point on the apache commons usage. Honestly, didn't even realised it's from there, I am also used to FileUtils so much...

Reply:
Thanks.  Made the changes.

Reply:
@belugabehr You only changed it in one place. I only commented once, but there were many lines that were changed in the same way in the PR.